### ISSUE TITLE
Google Chrome says "Deprecated feature used" in Issues tab

### ISSUE THREAD

**Issue by salmonb** [sockjs/sockjs-client#646]
hi,

When using SockJS in Chrome, it says that sockjs.js uses a deprecated feature at line 3285, which is:

```javascript
      global.addEventListener(event, listener, false);
```

and a breakpoint at this line shows that `global = window` and `event = 'unload'`. This is indeed a [deprecated feature](https://developer.mozilla.org/en-US/docs/Web/API/Window/unload_event). 

Can you please fix this?

**Comment by wr9dg17** [comment]
I have same problem, can you find the solution? @salmonb

**Comment by salmonb** [comment]
No, only the contributors of this project can solve this problem (I'm not). I opened this issue just to report the problem, and I hope they will fix it at some point.

It's not that annoying for the final users, but when you deliver an app that logs this, it doesn't look very professional...

**Comment by klwemu** [comment]
Looks like this mayor issue will not be fixed. Bad case.

**Comment by auvipy** [comment]
I will be happy to accept a PR to fix this
